ZFSin/zfs/module/zfs/zfs_ioctl.c

```diff
@@ -344,6 +357,9 @@ NTSTATUS zpool_get_size_stats(PDEVICE_OBJECT DeviceObject, PIRP Irp, PIO_STACK_L
 	if (!zstats)
```

Comment:
Can you merge the two if-statements instead of failing the call? That way I can avoid setting an empty string by passing a NULL as input buffer